### PR TITLE
feat: Disable data collection based on sessionId

### PR DESCRIPTION
**Pull request by denessapi**

### CODE REVIEW

File: trap/src/main/java/com/cursorinsight/trap/TrapManager.kt [hunk 1]
```diff
@@ -358,7 +368,7 @@ class TrapManager internal constructor(
         }
     }
 
-    override fun onActivityCreated(activity: Activity, savedInstanceState: Bundle?) {
+     override fun onActivityCreated(activity: Activity, savedInstanceState: Bundle?) {
```
Comment (gbence) [inline trap/src/main/java/com/cursorinsight/trap/TrapManager.kt]: Unnecessary space here

**Review by gbence — COMMENTED** [review]
The code looks good, I only have a whitespace related comment.

File: trap/src/main/java/com/cursorinsight/trap/TrapManager.kt (hunk 1 above)
Comment (denessapi) [reply trap/src/main/java/com/cursorinsight/trap/TrapManager.kt]: I've force pushed a new version with the fix.

**Comment by denessapi** [comment]
I've fixed it

**Review by gbence — APPROVED** [review]

**Pull request merged**